### PR TITLE
Revert "remove Fresh Prince"

### PR DESCRIPTION
**Pull request by nightpool**
Reverts new-xkit/XKit#1813

### CODE REVIEW

**Comment by AprilSylph** [comment]
why

**Comment by nightpool** [comment]
@AprilSylph WIP to remind me to add support for the new dashboard

File: Extensions/xkit_patches.js [hunk 1]
```diff
@@ -184,6 +184,16 @@ XKit.extensions.xkit_patches = new Object({
 
 	patches: {
 		"7.9.2": function() {
+			XKit.tools.cartesian_product = (items, current = []) => {
```
Comment (hobinjk) [inline Extensions/xkit_patches.js:187]: I get that a cartesian product is a defined math term but I don't think anyone else is going to use this function without a plain english description and a few jsdoc type annotations

File: Extensions/xkit_patches.js [hunk 2]
```diff
@@ -634,20 +683,41 @@ XKit.extensions.xkit_patches = new Object({
...
+				/**
+				 * Given any number of keys, turns them into a css selector
+				 * where key[0] contains key[1], etc.
+				 *
+				 * Because CSS commas always take precedence over spaces, and
+				 * :is has poor browser support, this requires enumerating all
+				 * possible combinations of `key[0] key[1]` and then separating
+				 * them with commas.
+				 *
+				 * @param {Array<String>} keys - the cssMap keys to combine.
```
Comment (hobinjk) [inline Extensions/xkit_patches.js:711]: As far as I can tell from a quick search the more common way of doing rest arguments is `{...String}`

**Pull request merged**